opencensus/trace/internal/grpc_trace_bin.cc

```diff
@@ -44,6 +44,7 @@ constexpr int kTraceOptionsOfs = kSpanIdOfs + 1 + kSpanIdLen;
 
 constexpr int kTotalLen =
     1 + 1 + kTraceIdLen + 1 + kSpanIdLen + 1 + kTraceOptionsLen;
+static_assert(kTotalLen == kGRPCTraceBinHeaderLen, "header length is wrong");
```

Comment:
instead of declaring kTotalLen use kGRPCTraceBinHeaderLen everywhere but keep the static_assert

Reply:
Done.